bbp/viz/rtneuron/default.nix

```diff
@@ -40,7 +40,7 @@ stdenv.mkDerivation rec {
   src = fetchgitPrivate {
     url = config.bbp_git_ssh + "/viz/RTNeuron";
     rev = "47dce1e11a6534dc2eb4971a589cec459c9cc274";
-    sha256 = "04215jjjl1sqc6lna19bd8zbyl9r89snjgysp7iwrmd9mj5x53wl";
+    sha256 = "14215jjjl1sqc6lna19bd8zbyl9r89snjgysp7iwrmd9mj5x53wl";
```

Comment:
Hey @tribal-tec 
This is this change that breaks the build. Can you revert it please?

Reply:
ah sure, this shouldn't be here